src/components/HTMLEngineProvider/HTMLRenderers/PreRenderer/index.js

```diff
@@ -30,8 +30,8 @@ class PreRenderer extends React.Component {
     scrollNode(event) {
         const node = this.ref.getScrollableNode();
         const horizontalOverflow = node.scrollWidth > node.offsetWidth;
-
-        if ((event.currentTarget === node) && horizontalOverflow) {
+        const isVerticalScrolling = Math.abs(event.deltaY) > 3; // This is for touchpads sensitive
+        if ((event.currentTarget === node) && horizontalOverflow && !isVerticalScrolling) {
```

Comment:
```suggestion
        if ((event.currentTarget === node) && horizontalOverflow &&  event.deltaX !== 0) {
```
What if only do this? What benefit do we get from checking scroll sensitivity?

Reply:
Ok, let me test this behavior.

Reply:
@parasharrajat and if we only do this `event.deltaX !== 0` the scroll will not be smooth. Tested

Reply:
Ok, I tested it. Looking fine to me. Add a comment instead `This is for touchpads sensitive` like you did https://github.com/Expensify/App/pull/9026/files#r873936354

Reply:
@parasharrajat More detailed comment added. PR is ready 👍